src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,59 @@
+package duke.task;
+
+/**
+ * Implements task objects.
+ *
+ * @author Audrey Felicio Anwar
+ */
+public class Task {
+    protected String description;
+    protected boolean done;

```

Comment:
Would it be better to name this isDone?

Reply:
Thanks for the suggestion. Yes, I think it would also be better if it is changed to `isDone` as it is a clearer boolean name. Will change this in future commits.